peachjam/models/settings.py

```diff
@@ -47,5 +47,15 @@ def __str__(self):
         return "Settings"
 
 
+class PeachjamLabels(models.Model):
+    name = models.CharField(max_length=1024)
+    value = models.CharField(max_length=1024)
+    setting = models.ForeignKey(PeachJamSettings, on_delete=models.CASCADE)
```

Comment:
@longhotsummer I"ve added a PeachjamLabels Model with fk to PeachjamSettings. Was this what we were going for?